agent/api/task.go

```diff
@@ -242,34 +242,34 @@ func (task *Task) UpdateMountPoints(cont *Container, vols map[string]string) {
 	}
 }
 
-// updateTaskKnownState updates the given task's status based on its container's status.
+// updateTaskKnownStatus updates the given task's status based on its container's status.
 // It updates to the minimum of all containers no matter what
-// It returns a TaskStatus indicating what change occured or TaskStatusNone if
+// It returns a TaskStatus indicating what change occurred or TaskStatusNone if
 // there was no change
+// Invariant: task known status is the minimum of container known status
 func (task *Task) updateTaskKnownStatus() (newStatus TaskStatus) {
-	llog := log.New("task", task)
-	llog.Debug("Updating task")
+	seelog.Debug("Updating task %v", task)
```

Comment:
The prior code dumped the whole struct.  There's value in it to see what's being updated, but I could also be convinced to make it more targeted if you felt strongly about it.